.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -0,0 +1,101 @@
+name: Merge Python Requirements Upgrade PRs
+
+on:
+  schedule:
+    - cron: "0 7 * * *"
```

Comment:
A link to cron.guru will help people understand this notation.

Reply:
Thanks, added.